bluebird-promisell.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const Promise = require('bluebird');
+const P = require('bluebird-promisell');
+const R = require('ramda');
+const S = require('sanctuary');
+
+//    data Text = Buffer | String
+//    readFile :: String -> String -> Promise Error Text
+const readFile = R.curry((encoding, filename) =>
+  new Promise((resolve, reject) => {
+    fs.readFile(filename, {encoding: encoding}, (err, data) => {
+      if (err) {
```

Comment:
Let's use `if (err != null)` here to avoid type coercion. I'm sorry that I didn't spot this earlier.